src/openrct2/ride/RideData.h

```diff
@@ -295,6 +296,8 @@ constexpr const RideTypeDescriptor DummyRTD =
     SET_FIELD(DefaultPrices, { 20, 20 }),
     SET_FIELD(DefaultMusic, MUSIC_STYLE_GENTLE),
     SET_FIELD(PhotoItem, SHOP_ITEM_PHOTO),
+    SET_FIELD(Properties, {0,0,0,0,0,0}),
```

Comment:
Format all of these in the same way other structs are, with spaces after { and before } and with one space after each comma. E.g.:

```suggestion
    SET_FIELD(Properties, { 0, 0, 0, 0, 0, 0 }),
```